pkgs/development/interpreters/jimtcl/default.nix

```
@@ -27,15 +30,16 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [
     pkg-config
     asciidoc
-  ];
+  ] ++ (lib.optional (stdenv.buildPlatform != stdenv.hostPlatform) buildPackages.jimtcl);
```

```suggestion
  ] ++ lib.optionals (stdenv.buildPlatform != stdenv.hostPlatform) [ 
    jimtcl
  ];
```

Splicing already gets us the buildPackages version


All the parentheses added in this pr are unnecessary